test/release/examples/primers/slices.chpl

```diff
@@ -93,26 +108,32 @@ proc main() {
   writeln();
 }
 
-// The iterator blockIter yields a range that defines a subset
-// of vec, which is always of length blk.
+/*
+  The iterator ``blockIter`` yields a range that defines a subset
+  of ``vec``, which is always of length ``blk``.
+*/
 iter blockIter(vec:range,blk) {
   for i in vec by blk {
     if (i + blk-1 <= vec.high) then
       yield i..i+blk-1;
   }
 }
 
-// This procedure sets each element of A to be the value of its
-// row index.
+/*
+  This procedure sets each element of ``A`` to be the value of its
+  row index.
+*/
 proc initBlock(A) {
   for (i,j) in A.domain {
     A(i,j) = i;
   }
 }
 
-// This procedure requires the domain of the array argument to
-// be [1..blk,1..blk].  It sets each element of A to be the value of
-// its row index.
+/*
+  This procedure requires the domain of the array argument to
+  be ``{1..blk,1..blk}``.  It sets each element of ``A`` to be
+  the value of its row index.
+*/
 proc initBlock2(A: [1..blk,1..blk]) {
   for (i,j) in A.domain {
     A(i,j) = i;
```

Comment:
Can you add a ` // main` on the line with the last closing brace, so it doesn't look as confusing for the rendered html?